migrating_from_ocp_3_to_4/premigration-checklists.adoc

```diff
@@ -0,0 +1,90 @@
+[id="premigration-checks"]
+= Premigration checklists
+include::modules/common-attributes.adoc[]
+:context: premigration-checks
+
+toc::[]
+
+Before you migrate your application workloads with the {mtc-full} ({mtc-short}), review the following checklists.
+
+[id="source-cluster-checklist_{context}"]
+== Source cluster checklist
+
+* [ ] The cluster meets the link:https://docs.openshift.com/container-platform/3.11/install/prerequisites.html#hardware[minimum hardware requirements].
+* [ ] The {product-title} version is 3.7, 3.9, 3.10, 3.11.
+* [ ] The {mtc-short} version is the same on all clusters.
+* [ ] All nodes have an active {product-title} subscription. This will avoid issues if you require customer support.
+* [ ] All the link:https://docs.openshift.com/container-platform/3.11/day_two_guide/run_once_tasks.html#day-two-guide-default-storage-class[run-once tasks] have been performed.
```

Comment:
* All the -> All of the ? (Same elsewhere in the file if you do change it)
* Is the link supposed to be jumping down to just the storage class section, or should it be at the top so that they do all of the sections in this assembly?